File: Order-Processing-Frontend/src/app/models/Author.ts
export class Author {
  first_name: string;
  last_name: string;
  address: string;
  phone_number: string;
  author_id: number;

  constructor(){
    this.first_name = "";
    this.last_name="";
    this.address = "";
    this.phone_number = "";
    this.author_id = -1;
  }
}

File: Order-Processing-Backend/src/main/java/csed/database/orderprocessingbackend/model/Author.java
package csed.database.orderprocessingbackend.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.io.Serializable;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Author {
    private Long author_id;
    private String first_name;
    private String last_name;

    private String address;
    private String phone_number;

    @Override
    public String toString(){
        return "( " + "'"+first_name+"'"+", "+ "'"+last_name+"'" + ", " + "'"+address + "'" + ", " + "'"+phone_number +"'" +")";
    }
}
